src/vanilla/CodeGeneratorJv.cs

```diff
@@ -82,6 +82,25 @@ protected Settings Settings
 
             //Enums
             await WriteEnumJavaFiles(codeModel).ConfigureAwait(false);
+            
+            //XML wrappers
+            if (codeModel.ShouldGenerateXmlSerializationCached)

```

Comment:
If the XML wrappers now are created in each CodeGenerator implementation, then can you extract this to a protected method that is inherited by CodeGeneratorJva and CodeGeneratorJvaf instead of just having it be copied and pasted.

Reply:
Good idea. Yes.